client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -121,8 +121,18 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 		}
 	);
 
-	const generatedDesigns = allDesigns?.generated?.designs || [];
-	const staticDesigns = allDesigns?.static?.designs || [];
+	// Only display free designs if siteSlugOrId does not exist
+	function filterFreeDesigns( designs: Design[] | undefined ) {
```

Comment:
Added logic to hide premium designs/features since we don't want to lead users to checkout when the siteSlug query param does not exist.